azurerm/resource_arm_databricks_workspace.go

```diff
@@ -0,0 +1,172 @@
+package azurerm
+
+import (
+	"fmt"
+	"github.com/Azure/azure-sdk-for-go/services/databricks/mgmt/2018-04-01/databricks"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/hashicorp/terraform/helper/validation"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/response"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+	"log"
+)
+
+func resourceArmDatabricksWorkspace() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceArmDatabricksWorkspaceCreateUpdate,
+		Read:   resourceArmDatabricksWorkspaceRead,
+		Update: resourceArmDatabricksWorkspaceCreateUpdate,
+		Delete: resourceArmDatabricksWorkspaceDelete,
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
```

Comment:
(missed this in the review) - going to push a commit to add some validation to this too